### ISSUE TITLE
Add modelCard to Service definition

### ISSUE THREAD

**Issue by mrutkows** [CycloneDX/specification#268]
More and more "AI" services are being offered as endpoints (generation, summarization, analysis, etc.); it would a valid consideration to be able to declare which AI models were being used behind such services.  That is, these are not necessarily packaged or offered as "components" per-se.

**Comment by mrutkows** [comment]
Again, we should re-visit what makes a component definition and service definition unique as both will have similar information (shared) from a BOM-perspective (i.e., read common base type).

**Comment by stevespringett** [comment]
Agreed. You should be able to accomplish this two different ways today.

1) An external reference on the service that specifies the SBOM that makes up that service
2) An external reference on the service that specifies the model card, which can appear in the same BOM or in an external BOM.
